### PR TITLE
MAISTRA-1400: Add IOR to Pilot

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

**Comment by bison** [comment]
I haven't had time to read through this yet, but is there a good reason to put this into Pilot? In general, it would be great if we could keep any mostly self-contained components that are Maistra / OpenShift specific separate. I think we should be trying to shrink the diff between upstream and Maistra, otherwise rebases are going to continue to be a nightmare. Personally, I'd prefer a bit of extra complexity in our operator and deployment bits if it means a smaller diff from upstream in the core components.

**Comment by jwendell** [comment]
@bison I've written the rationale of putting this in pilot in MAISTRA-1400.

Also, if there are no objections from our side, I'm planning to upstream this as well.

**Comment by bison** [comment]
Ah, okay. If we're upstreaming this, I have no problem with it going into Pilot.

**Comment by dgn** [comment]
In the `make gen` commit there are a lot of whitespace, newline and import order changes - are those accidental? I'd prefer changes like that to happen in a separate PR

**Comment by jwendell** [comment]
I'll rebase/resubmit this PR. This PR is what motivated me to fix all lint and gen-chek failures in maistra/istio.
Now that everything is merged I can proceed with a rebase for this one. Thanks for reminding me.

File: operator/pkg/apis/istio/v1alpha1/values_types.proto [hunk 1]
```diff
@@ -538,6 +538,7 @@ message GlobalConfig {
 
   // Controls whether one central istiod is enabled.
   google.protobuf.BoolValue centralIstiod = 62;
+
```
Comment (brian-avery) [inline operator/pkg/apis/istio/v1alpha1/values_types.proto]: why change the proto?

**Review by brian-avery — APPROVED** [review]

File: operator/pkg/apis/istio/v1alpha1/values_types.proto (hunk 1 above)
Comment (jwendell) [reply operator/pkg/apis/istio/v1alpha1/values_types.proto]: yeah, this is a leftover from previous interaction. I've noticed it but will only remove it after this one gets approved - trying to make eventual changes in batch.

**Review by knrc — APPROVED** [review]

**Pull request merged**